### PR TITLE
Add kubevirt/containerized-data-importer presubmits to testgrid

### PR DESCRIPTION
**Pull request by maya-r**

### CODE REVIEW

File: github/ci/testgrid/gen-config.yaml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 dashboards:
+- name: kubevirt-containerized-data-importer
```
Comment (dhiller) [inline github/ci/testgrid/gen-config.yaml:2]: So you don't want to split dashboards for cdi as we do with kubevirt?

Comment (maya-r) [reply github/ci/testgrid/gen-config.yaml:2]: if you mean separate between periodics and presubmits - cdi doesn't seem to have periodics

Comment (awels) [reply github/ci/testgrid/gen-config.yaml:2]: Not right now, but we might in the future, I just added a post submit yesterday, and was considering a daily periodic while I did that. So it makes sense to prepare for that.

**Review by fgimenez — APPROVED** [review]
/approve

Looks good, thanks! 

Not sure how long will take config merger to update our config and show the new dashboard but the process is completely automated, will keep an eye on https://testgrid.k8s.io/kubevirt

**Pull request merged**